src/Microsoft.VisualStudio.Validation.Tests/VerifyTests.cs

```diff
@@ -64,6 +63,17 @@ public void NotDisposed()
         {
             Assert.Equal(typeof(object).FullName, ex.ObjectName);
         }
+
+        try
+        {
+            Verify.NotDisposed(false, "message");
+            Assert.False(true, "Expected exception not thrown.");
+        }
+        catch (ObjectDisposedException ex)
```

Comment:
You can simplify this and the blocks above, to get rid of the try/catch. `Assert.Throws` will return the caught exception, which you can then do your `Assert.Equals` on.

For example:

```c#
ObjectDisposedException ex = Assert.Throws<ObjectDisposedException>(() => Verify.NotDisposed(false, (object)null));
Assert.Equal(string.Empty, ex.ObjectName);
Assert.Equal("message", ex.Message);
```

Reply:
Yes, I noticed the same thing. There's quite a bit of pattern precedent in this file so I appreciate that @iron9light. I'll follow up with a subsequent change to use the shorter pattern.